lib/ethereumex.ex

```diff
@@ -1,7 +1,7 @@
 defmodule Ethereumex do
   use Application
-  alias Ethereumex.Counter
   alias Ethereumex.Config
+  alias Ethereumex.Counter
```

Comment:
Can add spaces between `aliases`, `imports` and `moduledoc`? Alos can please change the order to:
1. doc
2. use
3. import
4. alias

Reply:
done and enforced by credo!